TASK/TaskFollowPointRoute.md

```diff
@@ -5,8 +5,15 @@ ns: TASK
 
 ```c
 // 0x595583281858626E 0xB837C816
-void TASK_FOLLOW_POINT_ROUTE(Ped ped, float speed, int unknown);
+void TASK_FOLLOW_POINT_ROUTE(Ped ped, float speed, int routeMode);
 ```
+### Flags
```

Comment:
You don't need the `### Flags` here

the enum should be surrounded in fences/code blocks
\```c
// put enum here
\```

Reply:
You should also get rid of the previous comment thats in a code block and just leave the description they have